app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -127,14 +166,68 @@ const useStyles = makeStyles((theme) => ({
     alignSelf: "center",
     fontWeight: "bold",
   },
+  menu: {
+    boxShadow: theme.shadows[1],
+    minWidth: "12rem",
+  },
+  menuPopover: {
+    transform: "translateY(2rem)",
+  },
+  menuTitle: {
+    fontWeight: "bold",
+    position: "relative",
+    pointerEvents: "none",
+    "&:after": {
+      content: "''",
+      position: "absolute",
+      left: 16,
+      bottom: 0,
+      borderBottom: "1px solid black",
+      width: 32,
+    },
+  },
+  menuItem: {
+    width: "100%",
+  },
+  menuItemMessage: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  notificationCount: {
+    color: grey[500],
+    fontWeight: "bold",
+  },
+  menuBtn: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    border: `1px solid ${grey[300]}`,
+    borderRadius: "35%",
+    backgroundColor: grey[200],
+    padding: 4,
+    cursor: "pointer",
+  },
+  avatar: {
+    height: 30,
+    width: 30,
+  },
```

Comment:
Use the `rem` equivalent for this so it scales in and out better when the user change default font size